src/clients/nfs-client4part.rb

```diff
@@ -54,6 +56,29 @@ def main
 
       nil
     end
+
+  private
+
+    # Generates the UI that allows to manage the NFS shares entries
+    #
+    # @return [Yast::Term] a term defining the UI
+    def create_ui
+      Wizard.SetHelpText(@help_text1)
+
+      ReplacePoint(id, FstabTab())
+    end
+
+    # Updates the UI that allows to manage the NFS shares entries
+    #
+    # @return [Boolean] true when entries are successfully replaced; false otherwise.
+    def refresh_ui
+      UI.ReplaceWidget(id, FstabTab())
+    end
+
+    # Returns the id for the NFS shares UI replace point
+    def id
+      @id ||= Id(:fstab_rp)
```

Comment:
NP: Yes, these old client classes are very weird. I am not sure if `#id` could be meaningful for them (who knows). I would avoid that method name. What about `ui_id`?